bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -316,23 +323,13 @@ default String getSrcset() {
     }
 
     /**
-     * Returns the value for the {@code width} html attribute of the image.
-     *
-     * @return the value for the image's {@code width} attribute, if one was set, or {@code null}.
-     * @since com.adobe.cq.wcm.core.components.models 12.21.0;
-     */
-    default String getWidth() {
-        return null;
-    }
-
-    /**
-     * Returns the value for the {@code height} html attribute of the image.
+     * Returns {@code new Dimension(width, height)} containing the base DAM asset's width and height (the DAM asset which is used as the {@code src} html attribute of the image).
      *
-     * @return the value for the image's {@code height} attribute, if one was set, or {@code null}.
-     * @since com.adobe.cq.wcm.core.components.models 12.21.0;
+     * @return {@code new Dimension(width, height)} containing the base DAM asset's width and height, if the image exists and is a DAM asset, or {@code new Dimension(0, 0)}.
+     * @since com.adobe.cq.wcm.core.components.models 13.0.0;
      */
-    default String getHeight() {
-        return null;
+    default Dimension getOriginalDimension() {
```

Comment:
If the expectation is that this will always return a dimension it should be annotated as not null